srcs/juloo.keyboard2/KeyModifier.java

```diff
@@ -281,7 +281,62 @@ private static String apply_fn_char(char c)
       case 'ס': return "segol";
       case 'ב': return "hataf_segol"; // reduced segol
       case 'צ': return "tsere";
-      // other
+      // Devanagari symbols
```

Comment:
So I'm using the `fn` key to allow a second "layer". Before, I tried to squeeze all the symbols into a single layer (kinda like the previous hindi layout), but I didn't like how congested/squeezed it turned out.

So for my layout, the original, non-`fn` layer contains the most-used characters, and the `fn` layer contains seldom used characters